docusaurus/docs-internals/editor-controls/multivalue-controls/motion.md

```diff
@@ -190,3 +253,132 @@ An array of animation names that should be disabled. In this case, you cannot se
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `motion` control that has the values for `"rotate"` and `"speed"` enabled by default and `"rotate"` is active.
+`"rotate"` has `"speed"` = `13` and `"direction"` = `"right"`, `"scale"` has `"speed"` = `7`.
+
+```js
+{
+  id: "motion", 
+  type: "motion",
+  default: {
+    active: "rotate",
+    rotateSpeed: 0.13,
+    rotateDirection: "right",
+    rotateEnabled: true,
+    scaleEnabled: true,
+    scaleSpeed: 7
+  }
+}
+```
+
+#### CSS examples
+
+Change the `.brz-text` element color with CSS using a `motion` control value.
+If `rotate` speed value >= 0.7, then we will colour `.brz-text` element to red, otherwise it will be blue.
+
+```js
+{
+  id: "motion",
+  type: "motion",
+  style: ({ value }) => {
+    if (value.rotate.speed >= 0.7) {
+      return {
+        "{{WRAPPER}} .brz-text": {
+          color: "red"
+        }
+      }
+    }
+
+    return {
+      "{{WRAPPER}} .brz-text": {
+        color: "blue"
+      }
+    }
+  }
```

Comment:
need to write more complex example with more used keys and more CSS properties because you have this possibility

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/5afdc14a-6138-48fe-981b-edf64c49300e)